src/python/pants/backend/python/register.py

```diff
@@ -92,6 +99,16 @@ def register_goals():
   task(name='unpack-wheels', action=UnpackWheels).install()
 
 
+def target_adaptors():
+  return {
+    'python_library': PythonTargetAdaptor,
+    'python_app': PythonAppAdaptor,
+    'python_tests': PythonTestsAdaptor,
+    'python_binary': PythonBinaryAdaptor,
+    'python_requirement_library': PythonRequirementLibraryAdaptor,
+  }
```

Comment:
I think you'll want to add this to each of the `register.py`s for each of the V2 linters. We specifically set those up so that they work without having `pants.backend.python` registered.

--

Actually, how do you feel about moving these back into `engine_initializer.py`? I think that we always want these registered, at least for the time being before we have the Target API. 

I think the intent here was cleanup, but if, instead, it was to test that this new mechanism works, consider adding a dedicated test.